### DISCUSSION TITLE
'Require pull-request reviews before merging' option not wooing up in gogs hosted instance

### DISCUSSION THREAD

**Discussion by srinivaschary** [gogs/gogs#7319] [Q&A]
I recently installed gogs on my local instance. Trying to explore the Pull Requests and Reviews, but I couldn't find the option in protect branch settings to manage number of required reviews. Is this features allowed in enterprise edition or does it need additional plugins/modules to be installed on the open source gogs installation?

**Comment by dmoralesdselanusa** [comment]
did you find out how?

**Comment by derak-kilgo** [comment]
First your repo needs to be a part of an org and not a personal repo. 
(Repos under your user name are personal repositories.)

From the web ui, you can edit the settings for your repo.
Pick "Branches" in the left menu and select a branch to protect.

Once you've picked a branch to protect, then you will have more options on how merges should happen and who has permission to merge.

![Image](https://github.com/user-attachments/assets/b2b6c632-c3d6-4da3-814d-3ef1802e94b5)

**Reply by dmoralesdselanusa** [reply]
That option protect your branch for not being pushed directly, but when you create the PR you can inmediatly merge without reviews from other colaborators or an specific organization member, is there an option to add a number of reviews to approve the pr or a review of an specific organization member?

**Reply by derak-kilgo** [reply]
Not to that detail. You can setup a single user or group of users who have the merge privilege. No it doesn't require review before merge.
Make sure your users aren't all admins on the repo. Give them the lowest practical privileges.
